app/Process.php

```diff
@@ -162,19 +189,62 @@ protected function run(string $command) : int
         return proc_close($proc);
     }
 
+    public function getOutputBuffer(): string
+    {
+        return $this->getFileContents($this->outputFileName);
+    }
+
+    public function getErrorBuffer(): string
+    {
+        return $this->getFileContents($this->errorFileName);
+    }
+
+    private function getFileContents(string $filename): string
+    {
+        $output = '';
+        $handle = @fopen($filename, 'r');
```

Comment:
can't you just use `file_get_contents` here? and just store file/path somewhere to simplify logic?

Reply:
actually `tempnam` already returns path so should work